### PR TITLE
Improved secret envs and fixed secret name

### PR DESCRIPTION
**Pull request by oneilcin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,13 +32,33 @@ make
 ```
 
 ## Generating cluster, machine, and provider-components files.
+The generate-yaml.sh bash script uses an environmental to receive the master and node ssh private keys.
```
Comment (alejandroEsc) [inline README.md]: thanks for adding this!

File: clusterctl/examples/ssh/generate-yaml.sh [hunk 2]
```diff
@@ -61,22 +61,50 @@ fi
...
     exit 1
 fi
 
 if [ -z ${CLUSTER_PASSPHRASE+x} ]; then
-    echo "using empty pass phrase to private key"
+    echo "using empty cluster pass phrase to private key"
     CLUSTER_PASSPHRASE=""
 fi
 
+if [ -z ${NODE_CLUSTER_PRIVATE_KEY_PLAIN+x} ] && [ -z ${NODE_CLUSTER_PRIVATE_KEY_ENCODED+x} ]; then
```
Comment (alejandroEsc) [inline clusterctl/examples/ssh/generate-yaml.sh]: nice!

File: README.md (hunk 1 above)
Comment (oneilcin) [reply README.md]: no problem!

**Comment by alejandroEsc** [comment]
is this ok for merging?

**Comment by alejandroEsc** [comment]
I will assume it is.

**Pull request merged**